code_challenges/code_wars/multiples_of_3_and_5.py

```diff
@@ -0,0 +1,14 @@
+def multiples_of_3_and_5(n):
+    multiples = []
+    total = 0
+    for number in range(1, n):
+        if number % 3 == 0 or number % 5 == 0:
+            multiples.append(number)
+    for number in multiples:
```

Comment:
Educational comment not intended to block merging:

Take a look at how the builtin function [sum() works](https://docs.python.org/3/library/functions.html#sum)

Example usage here might be something like
```python
def multiples_of_3_and_5(n):
    return sum([num for num in range(1, n) if num % 3 == 0 or num % 5 == 0])
```

Also here is the list of all [builtins](https://docs.python.org/3/library/functions.html)